src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -300,6 +310,36 @@ void BeginConnection(Guid nodeId, int portIndex, PortType portType)
             }
         }
 
+        void BeginMultipleConnections(Guid nodeId, int portIndex, PortType portType)
+        {
+            if (portType == PortType.Input) return; //only handle multiple connections when the port selected is an output port
+            activeStartPort = null;
+            var node = CurrentWorkspace.GetModelInternal(nodeId) as NodeModel;
+            if (node == null) return;
+
+            PortModel portModel = node.OutPorts[portIndex];
```

Comment:
Since this portModel is the selected port, you can rename `protModel` variable to `selectedPort` for better understanding of the code.